src/daemon/rpmostreed-transaction-types.cxx

```diff
@@ -1521,6 +1521,13 @@ deploy_transaction_execute (RpmostreedTransaction *transaction,
             return FALSE;
         }
 
+      if (deploy_has_bool_option (self, "apply-live"))
+        {
+          g_autoptr(GVariantDict) dictv = g_variant_dict_new (NULL);
+          g_autoptr(GVariant) live_opts = g_variant_ref_sink (g_variant_dict_end (dictv));
+          rpmostreecxx::transaction_apply_live(*sysroot, *live_opts);
+        }
+
       if (deploy_has_bool_option (self, "reboot"))
```

Comment:
Should we add a validation check in `rpmostreed_transaction_new_deploy` to make sure the user isn't requesting `apply-live` and `reboot`?

Once we do that, here making it an `else if` makes it obvious they're mutually exclusive.

Reply:
Yep, fixed!

Reply:
Hmm, haven't seen any changes here -- forgot a `git add`?

Reply:
Ah heh no I actually thought you were talking about the *client* code which had a similar problem, which I fixed.  I also now fixed the server side.